packages/shared/lib/contexts/governance/stores/latest-governance-transaction-ids.store.ts

```diff
@@ -0,0 +1,18 @@
+import { writable } from 'svelte/store'
+
+export const latestGovernanceTransactionIds = writable<{ [accountId: number]: string }>({})
```

Comment:
I think this name is confusing as it could mean the `latestGovernanceTransactionIds` that were confirmed. IMO `pendingGovernanceTransactionIds` would be more accurate since one it is confirmed then it is removed, meaning it's no longer pending.

Reply:
Makes sense